packages/blockchain-extension/extension/commands/addGatewayCommand.ts

```diff
@@ -48,12 +49,12 @@ export async function addGateway(): Promise<{} | void> {
 
         if (gatewayMethod === UserInputUtil.ADD_GATEWAY_FROM_ENVIRONMENT) {
 
-            const environments: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll(false, false, true);
+            const environments: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll([], [EnvironmentFlags.ANSIBLE]);
```

Comment:
Does this exclude all Ansible environments?

Reply:
yes